data/api/client-server/profile.yaml

```diff
@@ -67,163 +120,144 @@ paths:
       tags:
         - User data
     get:
-      summary: Get the user's display name.
+      summary: Get a profile field for a user.
```

Comment:
This needs an `x-changedInMatrixVersion` key that explains the change of the endpoint.